### PR TITLE
Zip code entry and validation

### PR DESCRIPTION
**Pull request by erinkav**
## Summary
Zip code entry and validation page for users who opt not to allow location tracking. Find this page by clicking "Enter my zip code instead" from the Location page

## Screenshots
Screen Shot 2020-09-16 at 11 18 44 PM
Screen Shot 2020-09-16 at 11 18 34 PM

## Review notes
- I'm unfamiliar with React Native norms please let me know if there are conventions I'm missing
- Would be great to get a shared stylesheet going or shared button and input components. I'm opening this one now to allow the fonts to be shared but can take splitting a shared button and input component out later
- Next PR will be for restyling the Location page and the phone entry page

### CODE REVIEW

File: pages/EnterZipCode/index.js [hunk 1]
```diff
@@ -1,19 +1,80 @@
...
+      <Text style={styles.text}>Let's start with your location</Text>
+      <Input inputStyle={styles.input}
+        onFocus={() => setErrorMessage('')}
+        ref={input}
+        placeholder={'Enter your zip code'}
+        placeholderTextColor="#CDCDCD80"
+        inputContainerStyle={styles.inputContainer}
+        errorMessage={errorMessage}
+        placeholderTextColor="#CDCDCD80"
+        onChangeText={(zip) =>  setZipCode(zip) }/>
```
Comment (felipellrocha) [inline pages/EnterZipCode/index.js:75]: Anyway we can convert this into a state shortname? It's what I'll need later in the flow

File: pages/EnterZipCode/index.js [hunk 2]
```diff
@@ -1,19 +1,80 @@
...
+      <Button title='Set my location' buttonStyle={styles.button} onPress={() => {
+          if(isValidZipCode(zipCode)) {
```
Comment (felipellrocha) [inline pages/EnterZipCode/index.js:77]: nit.: Can we extract this method into a useCallback? I have a rule of thumb, if you need more than one line to build your handler function, it's probably better to extract it.

File: pages/EnterZipCode/index.js [hunk 3]
```diff
@@ -1,19 +1,80 @@
-import React from "react";
+import React, {useState } from "react";
 import { View, Text } from "react-native";
-import { Input } from "react-native-elements";
+import { Input, Button } from "react-native-elements";
+
+const styles = {
```
Comment (felipellrocha) [inline pages/EnterZipCode/index.js:6]: Thank you! 🙏

**Review by felipellrocha — APPROVED** [review]
lgtm! 😄

File: pages/EnterZipCode/index.js (hunk 1 above)
Comment (erinkav) [reply pages/EnterZipCode/index.js:75]: oh good to know! i found this stack overflow with an example of how to match zip codes to states wihtout calling an external API. i can extract that into a helper function. I'm going to do that in a separate PR along with the "Allow location sharing" so I can use the state matcher method on both. So assume you'll have the state and i'll get that up as soon as i can https://stackoverflow.com/questions/28821804/how-can-i-quickly-determine-the-state-for-a-given-zipcode

File: pages/EnterZipCode/index.js (hunk 2 above)
Comment (erinkav) [reply pages/EnterZipCode/index.js:77]: yes! had the same intention but forgot to come back to this

**Pull request merged**